packages/core/admin/index.js

```diff
@@ -92,17 +92,30 @@ async function build({ plugins, dir, env, options, optimize }) {
 async function createPluginsJs(plugins, dest) {
   const pluginsArray = plugins.map(({ pathToPlugin, name }) => {
     const shortName = _.camelCase(name);
+
+    /**
+     * path.join, on windows, it uses backslashes to resolve path.
+     * The problem is that Webpack does not windows paths
+     * With this tool, we need to rely on "/" and not "\".
+     * This is the reason why '..\\..\\..\\node_modules\\@strapi\\plugin-content-type-builder/strapi-admin.js' was not working.
+     * The regexp at line 105 aims to replace the windows backslashes by standard slash so that webpack can deal with them.
+     * Backslash looks to work only for absolute paths with webpack => https://webpack.js.org/concepts/module-resolution/#absolute-paths
+     */
+    const realPath = path
+      .join(path.relative(path.resolve(dest, 'admin', 'src'), pathToPlugin), 'strapi-admin.js')
+      .replace(/\\/g, '/');
```

Comment:
If we want to always have `/` even on Windows I think you can use `path.posix.join()`. This will force the POSIX separator which is `/`.

To avoid having to add `posix` to all the path methods you could do an `const { posix: path } = require('path');` or even `const path = require('path').posix;`. This should use `/` instead of `\\` on windows.

Reply:
THis should be in my PR... I'll clean this up